ansible/mnesia_reset_once.yml

```diff
@@ -6,26 +6,59 @@
   vars:
     project_root: "{{ ansible_env.HOME }}/node"
     aeternity_bin: "{{ project_root }}/bin/aeternity"
+    public_ipv4: "{{ ansible_ec2_public_ipv4|default(ansible_ssh_host)|default(ansible_host)|default(inventory_hostname) }}"
+    datadog_api_key: "{{ lookup('hashi_vault', 'secret=secret/datadog/deploy:api_key') }}"
+    datadog_app_key: "{{ lookup('hashi_vault', 'secret=secret/datadog/deploy:app_key') }}"
 
   tasks:
-    - name: Get nodes status
-      uri:
-        url: "http://{{ item }}:{{ node_config.http.external.port | default('3013') }}/v2/status"
-      register: hosts_status
-      loop: "{{ play_hosts }}"
-      delegate_to: localhost
-      run_once: true
+    - block:
+      - name: Get nodes status
+        uri:
+          url: "http://{{ item }}:{{ node_config.http.external.port | default('3013') }}/v2/status"
+        register: hosts_status
+        loop: "{{ play_hosts }}"
+        delegate_to: localhost
 
-    - name: Fail if no synced node found
-      fail:
-        msg: No synced node
-      delegate_to: localhost
-      when: >
-        hosts_status.results
-        | map(attribute='json')
-        | selectattr('sync_progress', 'equalto', 100)
-        | list
-        | length == 0
+      - name: Fail if no synced node found
+        fail:
+          msg: No synced node
+        delegate_to: localhost
+        when: >
+          hosts_status.results
+          | map(attribute='json')
+          | selectattr('sync_progress', 'equalto', 100)
+          | list
+          | length == 0
+
+      - name: Send Datadog event
+        datadog_event:
+          host: "{{ public_ipv4 }}"
+          alert_type: success
+          title: Reset mnesia database
+          text: "Synced node found"
```

Comment:
This is not very useful, I'd rather move it below after the actual node reset. Btw I just noticed we missed the usual node health checks.

In general one success notification should be sent once the full playbook completes (with additional health checks).
A failing notification should be sent regardless of the error/fail reason (of any task).
Nice to have is the actual failing task and reason carried in `ansible_failed_task` and `ansible_failed_result`
 (see the bottom of https://docs.ansible.com/ansible/latest/user_guide/playbooks_blocks.html)